src/main/services/modOrganizer.service.ts

```diff
@@ -47,6 +48,10 @@ export class ModOrganizerService {
     return (await psList()).filter(ModOrganizerService.filterMO2).length > 0;
   }
 
+  async initMO2Checks() {
+    setInterval(async () => (this.mo2Running = await this.isRunning()), 1000);
```

Comment:
It's normally better to use `SetTimeout` and have a recursive function. That way, if the previous execution hasn't finished, it will not call it again.
With `SetInterval`, it could end up calling again before the previous has finished.

Something like

```suggestion
setTimeout(()=> {
    this.mo2Running = await this.isRunning();
    this.initMO2Checks();
})